src/test/java/org/opensrp/service/multimedia/ObjectStorageMultimediaFileManagerTest.java

```diff
@@ -0,0 +1,64 @@
+package org.opensrp.service.multimedia;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.opensrp.repository.MultimediaRepository;
+import org.opensrp.service.ClientService;
+import org.springframework.beans.factory.annotation.Value;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+
+public class ObjectStorageMultimediaFileManagerTest extends BaseMultimediaFileManagerTest {
```

Comment:
Maybe you should pass `@ActiveProfiles` so that one of S3 or AliCloud profile is used